src/syscall/aarch64-linux.cr

```diff
@@ -2,7 +2,7 @@
 
 module Syscall
   # Based on https://github.com/torvalds/linux/blob/master/include/uapi/asm-generic/unistd.h
-  private enum Code : UInt64
+  enum Code : UInt64
```

Comment:
This enum is internal, so it should be `nodoc`, right?

Reply:
Fixed.